main/2020/CG-02.md

```diff
@@ -980,6 +980,7 @@ RT: What's the indirection you're referring to.
 BT: So suppose you have class A, some java library two fields. User library imports the type with one field. Now you compile the user module offline, it has an extra indirection, it doesn't know where the field is at. A table of numbers that you have to lookup through, another load.
 
 RT: Another way to do it is talking to other communities: two other ways 
+RT: Another way to do it is talking to other communities: two other ways
```

Comment:
duplicated line

Reply:
Oops, yeah, my editor was misconfigured and I noticed after the fact that it had stripped all the white space. I thought I undid all the unwanted changes, but I guess I missed half of one, heheh. Fixed in the forced push.